.nvmrc

```diff
@@ -1 +1 @@
-20
+20.19.5
```

Comment:
Probably should update the docs in the README as well as any CI workflows that set up node to align with this version but don't read from this file.